development/extensions/skeleton_extension.rst

```diff
@@ -30,17 +30,17 @@ installed into a phpBB board just the same as any other.
 
 .. note::
 
-    The Skeleton Extension is a tool for extension developers. As such,
-    it is intended to be installed in a phpBB board used for
-    offline/local development of extensions. It is not necessarily
-    intended to be installed on live/production web sites.
+    The Skeleton Extension is intended to be installed in a phpBB
+    board used for offline/local development of extensions. It is
+    not necessarily intended to be installed on live/production web
```

Comment:
https://github.com/phpbb-extensions/phpbb-ext-skeleton/issues/46
Yes, indeed, but. Any robot crawling there can hit the link etc.. etc.. or am I wrong?
I think it would be safe to have those perms.